cpp/CMakeLists.txt

```diff
@@ -220,7 +220,10 @@ if(BUILD_CUML_TESTS OR BUILD_PRIMS_TESTS)
   find_package(Threads)
 endif()
 
+# thrust before rmm, rmm before raft so we get the right version of thrust/rmm
```

Comment:
Does the order of libcudacxx matter? Will it start to matter once we switch to CCCL 2?

Reply:
It must be before RMM, and I've already simplified this (and improved the comment text) in some local commits for CCCL 2.

Reply:
OK got it, thanks.